docs/pull_requests.md

```diff
@@ -0,0 +1,89 @@
+# Pull Request process
+
+Pull Requests in the `openshift-ansible` project follow a
+[Continuous](https://en.wikipedia.org/wiki/Continuous_integration)
+[Integration](https://martinfowler.com/articles/continuousIntegration.html)
+process that is similar to the process observed in other repositories such as
+[`origin`](https://github.com/openshift/origin).
+
+Whenever a
+[Pull Request is opened](../CONTRIBUTING.md#submitting-contributions), some
+automated test jobs must be successfully run before the PR can be merged.
+
+Some of these jobs are automatically triggered, e.g., Travis and Coveralls.
+Other jobs need to be manually triggered by a member of the
+[@team-openshift-ansible-contributors](https://github.com/orgs/openshift/teams/team-openshift-ansible-contributors) team.
+
+## Triggering tests
+
+We have two different Jenkins infrastructures, and, while that holds true, there
+are two commands that trigger a different set of test jobs. The objective is to
+move towards a single infrastructure, simplifying the workflow.
+
+- **Test jobs on the older infrastructure**
+
+  Team members can trigger the set of test jobs in the older infrastructure by
+  writing a comment with the exact text `aos-ci-test` and nothing else.
+  <!--
+    REVIEW: who exactly has permission to trigger this -- define "team" above
+    -- Is it all of team-openshift-ansible-contributors? Or a more general group?
+  -->
+
+  The Jenkins host is not publicly accessible. Test results are posted to S3
+  buckets when complete, and links are available both at the bottom of the Pull
+  Request page and as comments posted by
+  [@openshift-bot](https://github.com/openshift-bot).
+
+- **Test jobs on the newer infrastructure**
+
+  Team members can trigger the set of test jobs in the newer infrastructure by
+  writing a comment containing `[test]` anywhere in the comment body.
+
+  The [Jenkins host](https://ci.openshift.redhat.com/jenkins/job/test_pull_request_openshift_ansible/)
+  is publicly accessible. Like for the older infrastructure, the result of each
+  job is also posted to the Pull Request as comments and summarized at the
+  bottom of the Pull Request page.
+
+<!-- NOTE: https://ci.openshift.redhat.com/jenkins/job/openshift-ansible/ is not being used since Jan 18, 2017, we may clean it up -->
+
+## Triggering merge
+
+After a PR is properly reviewed and a set of required jobs reported
+successfully, it can be tagged for merge by one of the
+[@team-openshift-ansible-contributors](https://github.com/orgs/openshift/teams/team-openshift-ansible-contributors)
+team members by writing a comment containing `[merge]` anywhere in the comment
+body.
+
+Tagging a Pull Request for merge puts it in an automated merge queue. The
+[@openshift-bot](https://github.com/openshift-bot) monitors the queue and merges
+PRs that pass all of the required tests.
+
+### Manual merges
+
+The normal process described above should be followed: aos-ci-test and [test] /
```

Comment:
Can you wrap the commands in "code" markup?

> The normal process described above should be followed: `aos-ci-test` and `[test]` / `[merge]`

It will emphasize each item clearly.